src/pages/app/generic/DetailView.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 
 import Button from '../../../components/buttons/Button';
 import { ButtonGroup } from '../../../ui/styled/global';
-import { PageWrapper } from '../../../ui/styled/styles';
 import {
   FullDetailsWrapper,
   GrayWrapper,
   GridWrapper,
   HeadWrapper,
+  PageWrapper,
```

Comment:
Is change necessary?

Reply:
Yes actually, I think two different styles with similar names were conflicting so I had to swap it.